.github/workflows/paladin-PR-build.yml

```diff
@@ -50,54 +50,27 @@ jobs:
         run: ./gradlew -PcomposeLogs=true -PverboseTests=true --no-daemon --parallel --max-workers=5 build
   
   image-build:
-    runs-on: ubuntu-latest
-    env:
-      IMAGE_REGISTRY: kaleido.io  # just a placeholder
-      IMAGE_NAME: paladin
-      IMAGE_TAG: test  # just a placeholder, we should replace this with the workflow run number
-      BUILD_PLATFORM: linux/amd64
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-          submodules: recursive
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
+    # run only on PRs
+    if: github.event_name == 'pull_request'
+    uses: ./.github/workflows/build-image.yaml
+    with:
+      dockerfile: ./Dockerfile
+      registry: paladin.io
```

Comment:
What is this registry?

Reply:
it's a mock registry, this image is not pushed.

Reply:
?? does the workflow not fail?

Reply:
No, in this WF the images are not pushed to the registry, it's just testing the image build process